pkg/config/validation/validation.go

```diff
@@ -2054,8 +2065,26 @@ var ValidateAuthorizationPolicy = registerValidateFunc("ValidateAuthorizationPol
 					}
 				}
 			}
+
+			if !tcpRulesInFrom && !tcpRulesInTo && ((rule.To != nil && len(rule.To) != 0) ||
+				(rule.From != nil && len(rule.From) != 0)) && in.Action == security_beta.AuthorizationPolicy_DENY {
+				ns := cfg.Meta.Namespace
+				if in.Selector != nil && in.Selector.MatchLabels != nil {
+					var label string
+					for key, val := range in.Selector.MatchLabels {
+						label = label + " " + key + ":" + val
+					}
+					warning = fmt.Errorf("all the traffic to TCP ports for pods with labels %s in"+
```

Comment:
nit: I would make warning: "configured AuthorizationPolicy will deny all traffic to TCP ports due to using only HTTP attributes in a DENY rule; it is recommended to explicitly specify the port"

We don't need the labels or namespace, the warning is associated with the resource already

Reply:
sure, thanks